geomesa-kafka/geomesa-kafka-08-datastore/src/main/scala/org/locationtech/geomesa/kafka08/KafkaProducerFeatureStore.scala

```diff
@@ -139,13 +139,13 @@ class KafkaFeatureWriterModify(sft: SimpleFeatureType, producer: Producer[Array[
 
 object KafkaProducerFeatureStoreFactory {
 
-  private [kafka] val FeatureIds = new AtomicLong(0L)
+  private [kafka08] val FeatureIds = new AtomicLong(0L)
 
   def apply(broker: String): FeatureSourceFactory = {
 
     val config = {
       val props = new ju.Properties()
-      props.put("metadata.broker.list", broker)
+      props.put(new KafkaUtils08().brokerParam(), broker)
```

Comment:
Another static method call.  I'll only note these for Kafka 8, but we can understand that they apply for all versions.